### DISCUSSION TITLE
Contributor call: 22 July 2023

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#4458] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This page will be updated after the call with notes.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: July 22, 2023 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jul-22-2023/4pm).
Or check the Events tab in Discord!
The meetings will always be at 6pm, Antwerp time.

### Agenda
👀

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @tangerineshark 
- @lexander2002
- @KaerMorhan 
- @BenJamesBen 
- @woutervdub 
- Braveness23
- Thrunic
- @SeaZeeZee 
- @darigovresearch 
- Jacquie Gibson

✨ **Thrunic** continues to work on _Shelly_ — one of the patterns that will be released with the V3.

- Wouter's comment: The new version is changing how pattern parts interact with each other so you don't have to change the code every time the source code changes (for example, if a pattern uses Bella body block to build off of).

✨ **Ben** continues to work on Eunice and Jane, ironing out mistakes that come to the surface, discusses cutting.

- Zwaluw's comment: When working with linen, using [pulling the thread method](https://youtu.be/7CjBlQ50UTA) is very handy to have a straight line to follow, it works especially well with historic patterns as people where trying to use as much of the fabric as possible.
- Wouter: It is not a viable technique with modern cotton though.
- Karen: Yes, but also you don't really have to worry about the grain when cutting quilt cotton because it is so dense. Speaking of cutting, do you cut on the floor or on the table?
- Zwaluw: The documentation for Jane is not fully complete yet because I want people to test it and see which way is more comfortable for them. Btw, here's also [a link to accessible theming](https://jane-git-fixingshit-seazeezee.vercel.app/), the light theme is done, dark theme is WIP.

✨ Discussing current projects, makers' spaces and dreams/plans for the future

- **Braveness23** is making a huge curtain for the back of the house and is restoring his front porch by recreating balusters from expandable foam with strengthening material in the middle. Also, showing off [local makers' space](https://hive13.org/spy/).
- **Zwaluw** is going on an archeological trip to Scotland. **Karen** discussing the possibility of making a field pants pattern.
- **Wouter** is making a chandelier, the pendants are already done, he's figuring out how to hang them.
- **Thrunic** is making a wetsuit but more stretchy.

✨ **Morgan** mentions the translation update ([#4539](https://github.com/freesewing/freesewing/discussions/4539)).

- Invite automation: now it is possible to receive the invite by email.
- Language coordinators: there will be person for each language that will be a single point contact for that language.
- DeepL integration: now DeepL will be used instead of the free Crowdin engine. All of the languages are fully translated, DeepL translations need to be proofread and corrected if necessary.
- Ukrainian is now a fully integrated language and has its own channel on Discord. The majority of the text is translated with DeepL, Morgan and @bbdapalkow are now in the process of proofreading and fixing it.
- Language-splitting the website: **Joost** ran some tests, for more info please visit the discussion thread mentioned above.

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
